src/api/src/config.rs

```diff
@@ -62,7 +72,15 @@ impl TryFrom<&Path> for Config {
     ///        # Only used in Remote Attestation-Service mode
     ///        "as_addr": "http://127.0.0.1:50004",
     ///        # Only used in Native Attestation-Service mode
-    ///        "as_config_file_path": "/etc/as-config.json"
+    ///        "as_config_file_path": "/etc/as-config.json",
+    ///        # Only used in Amber Attestation-Service mode
+    ///        "amber" : {
+    ///            "base_url": "https://amber.com",
+    ///            "api_key": "tBfd5kKX2x9ahbodKV1...",
+    ///            "certs_file": "/etc/amber/amber-certs.txt",
+    ///            # Optional, default is false.
+    ///            "allow_unmatched_policy": true
+    ///        }
```

Comment:
Could you also please update the `docs/config.md` file as well, now that we have one?

Reply:
> Could you also please update the `docs/config.md` file as well, now that we have one?

I have updated the `docs/config.md`